docs/fieldnorm/README.md

```diff
@@ -1,13 +1,20 @@
 # Introduction
 
-In the past years bioinformatics has become a crucial part of several medical laboratories, such as Genetics , Pathology and Medical Microbiology. Hence, bioinformaticians need to comply to both the ISO-15189 and the IVDR. 
+In the past years bioinformatics has become a crucial part of several medical laboratories, such as Genetics , Pathology and Medical Microbiology. 
+Hence, bioinformaticians need to comply to both the ISO-15189 and the IVDR. 
 
-However, the ISO-15189 does not include bioinformatic or software development processes. Fortunately, the IEC-62304 exists, which is a harmonized standard about "Medical device software – Software life cycle processes". The use of this standard remains voluntary, however, it gives good guidance to demonstrate compliance with legal requirements. 
+However, the ISO-15189 does not include bioinformatic or software development processes. 
+Fortunately, the IEC-62304 exists, which is a harmonized standard about "Medical device software – Software life cycle processes". 
+The use of this standard remains voluntary, however, it gives good guidance to demonstrate compliance with legal requirements. 
 
-We took the IEC-62304 as a guidance to create a fieldnorm for bioinformatics, to make sure that the software we create is developed and manufactured in accordance with the state of the art taking into account the principles of development life cycle, risk management, including verification and validation, as required by the IVDR (Annex I, point 16). 
+We took the IEC-62304 as a guidance to create a fieldnorm for bioinformatics, to make sure that the software we create is developed and manufactured in accordance with the state-of-the-art taking into account the principles of development life cycle, risk management, including verification and validation, as required by the IVDR (Annex I, point 16). 
```

Comment:
This correction is incorrect. State of the art is not hyphenated. See also https://en.wikipedia.org/wiki/State_of_the_art

Reply:
Ah thanks, I learned something new.

Reply:
I think I copied this from the IVDR